### DISCUSSION TITLE
[Feature] logging for shared albums

### DISCUSSION THREAD

**Discussion by athornfam2** [immich-app/immich#10055] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Bring better logging to each users profile that’s sharing an album. Know the IP that was used to access the service and make those that log into the album maybe make a name fixed to the IP so they are not typing a name in a bunch of times. Only when the address changes. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by corvy** [comment]
I agree. I would love to get better visibility in the logging, and also in the GUI for the use of shared links and shared albums.

**Comment by epifeny** [comment]
+1 on this. 
* More insights into public link usage.
* IP logging and potentially fixed names for public link visitors to see who accessed shared content.
